src/app/users/documents/list-documents.component.html

```diff
@@ -0,0 +1,37 @@
+<page-layout [ready]="data$ | async">
+  <ng-container *ngIf="data">
+    <page-content [mode]="'empty'" [heading]="i18n.document.title"
+      [mobileHeading]="i18n.document.mobileTitle"
+      [headingActions]="headingActions$ | async">
+    </page-content>
+
+    <results-layout [results]="data" [toLink]="toLink">
+      <table *resultTable class="table table-hover cursor-pointer">
+        <thead>
+          <th>{{ i18n.general.name }}</th>
+          <th>{{ i18n.general.description }}</th>
+          <th class="actions"></th>
+        </thead>
+        <tbody>
+          <tr *ngFor="let row of data"
+            (click)="showPrintButton(row.kind) ? print(row) : router.navigate(path(row)); $event.stopPropagation() ">
+            <td>{{ row.name }}</td>
+            <td>{{ row.description }}</td>
+            <td class="actions actions-multiple">
+              <button *ngIf="showPrintButton(row.kind)"
```

Comment:
The `showPrintButton` name is strange for me. It doesn't print, it downloads the document.
The main web shows a download icon, not print.
My advice is to rename the functions to `showPrintButton` to `isStatic` and `print` to `download`. Also, change the icon to `save_alt`.